utilities/provisioning_config_parser.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+import ansible.inventory
+
+
+def hosts_for_tag(tag):
+    hostfile = "provisioning_config"
+
+    if not os.path.isfile(hostfile):
+        print("File 'provisioning_config' not found at {}".format(os.getcwd()))
+        sys.exit(1)
```

Comment:
@tleyden what looks strange here? I am running through PEP8 in my IDE, I just sometimes ignore long lines. I can stop doing that going forward.